apps/api/src/server/app/repositories/appeal.repository.js

```diff
@@ -64,32 +65,26 @@ const appealRepository = (function () {
 			});
 		},
 		getById: function (
```

Comment:
This is now essentially the same signature as `getByIdIncluding`, so we should drop one of them (probably drop getByIdIncluding). However, we should use the signature from`getByIdIncluding` as the include keys are automatically generated by prisma based on the relations defined in the schema

Reply:
As I have mentioned before, I disagree. By defining the inclusions in the attributes themselves, we are leaking prisma code into the rest of the repo. Additionally, we are running the risk of repeating a lot of the same code in many places.